index.js

```diff
@@ -241,10 +201,7 @@ function resolve_array(arr) {
  * @return {String}
  */
 
-function type(v) {
-  return isArray(v)
-    ? 'array'
-    : v.vo
-    ? 'vo'
-    : typeof v;
+function typeOf(v) {
```

Comment:
let's keep it simple and remove this piece for now. if it comes up in an issue we can address it then.